draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -908,13 +948,292 @@ See stored query at "/contacts/stored-queries/42".
+        content:
+      </t>
+<artwork type="http-message">
+GET /stored-queries/4815162342 HTTP/1.1
+Host: example.org
+Date: Sun, 21, Sep 2025, 12:08:00 GMT
+Accept: text/csv
+If-Modified-Since: Sun, 31 Aug 2025, 00:00:00 GMT
+
+</artwork>
+      <t>
+        Here, the state of the data set indeed changed, so new content is returned:
+      </t>
+      <artwork type="http-message">
+HTTP/1.1 200 OK
+Date: Sun, 21, Sep 2025, 12:08:00 GMT
+Content-Type: text/csv
+Last-Modified: Thu, 18 Sep 2025, 19:56:00 GMT
+Vary: Accept-Query, Content-Encoding, Content-Type
+
+decade, total, with errata, % with errata, average page count
+1960, 26, 5, 19.2, 5.3
+1970, 666, 18, 2.7, 6.1
+1980, 376, 44, 11.7, 23.4
+1990, 1593, 269, 16.9, 25.5
+2000, 2888, 1048, 36.3, 27.3
+2010, 2954, 895, 30.3, 26.1
+2020, 1133, 230, 20.3, 26.2
+</artwork>
+      <t>
+        (Note the change in the row for this decade.)
+      </t>
+      <t>
+        The diagrams below illustrate the use of conditional requests and how they can differ
+        when a URI is assigned to the equivalent resource (and when the client is taking advantage
+        of it). The fictitious field name "Validator" is used for demonstration purposes.
+      </t>
+      <figure title="Data Flow with QUERY only">
+        <artset>
+          <artwork type="ascii-art" name="flow-query" align="center"><![CDATA[
+Client                             Resource
+|                                         |
+|                                         |
+| QUERY with content                      |
++---------------------------------------->|
+|                                         |
+|                                         |
+|                                  200 OK |
+|                          Validator: foo |
+|<----------------------------------------+
+|                                         |
+|                                         |
+| QUERY with content                      |
+| (conditional on 'foo')                  |
++---------------------------------------->|
+|                                         |
+|                                         |
+|                        304 Not Modified |
+|                          Validator: foo |
+|<----------------------------------------+
+|                                         |
+|                                         |
+|                                  +--------------+
+|                                  | State Change |
+|                                  +--------------+
+|                                         |
+| QUERY with content                      |
+| (conditional on 'foo')                  |
++---------------------------------------->|
+|                                         |
+|                                         |
+|                                  200 OK |
+|                          Validator: bar |
+|<----------------------------------------+
```

Comment:
```suggestion
|<----------------------------------------+
|                                         |
```

The tails on the lines are necessary to avoid making the diagram look a bit strange.